### ISSUE TITLE
Ambient occlusion uses TEXCOORD_1 instead of TEXCOORD_0

### ISSUE THREAD

**Issue by echadwick-wayfair** [mrdoob/three.js#24325]
**Describe the bug**
When both TEXCOORD_0 and TEXCOORD_1 are present, three.js will use TEXCOORD_1 for ambient occlusion. 

However TEXCOORD_0 is expected behavior for content authors, because [0 is the default in the spec](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#_material_occlusiontextureinfo_texcoord) when the author has not specified TEXCOORD_1 in the material via the texCoord property, nor via the [KHR_texture_transform](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_texture_transform/README.md) extension. 

Other common glTF renderers seem to use the default TEXCOORD_0 when rendering this example model.

**To Reproduce**
1. Download attached example ZIP file, and extract the GLB.
2. Load the GLB in https://threejs.org/editor/
3. Result: "Incorrect" text is shown for occlusion.

**Expected behavior**
"Correct" text should be shown for occlusion, if TEXCOORD_0 is being used as intended.

**Screenshots**
![screenshots](https://user-images.githubusercontent.com/40177036/177862770-aa3d7c14-3549-49cd-a0e9-4b0721b3bb28.jpg)
Screenshots from Visual Studio Code, with glTF Tools extension.

**Platform:**
 - Device: desktop
 - OS: Windows 10
 - Browser: Chrome
 - Three.js version: r142
 
***Example file***
[occlusiontest_gltf.zip](https://github.com/mrdoob/three.js/files/9067008/occlusiontest_gltf.zip)

Thanks for listening!

**Comment by WestLangley** [comment]
three.js built-in materials always use TEXCOORD_1 for AO and _light maps_,  and TEXCOORD_0 for everything else.

**Comment by donmccurdy** [comment]
Good point on lightmaps, yes — I've added a longer response at https://github.com/donmccurdy/glTF-Transform/discussions/610. We'd like to fix this of course but I'm not sure whether we'll have a 'dedicated solution' for the problem in the current material system, of if it will require the upcoming node-based material system to solve. Closing here since this is also tracked in https://github.com/mrdoob/three.js/issues/12608.
